src/AgGrid.elm

```diff
@@ -247,12 +247,19 @@ type Sorting
 -}
 type CellEditor
     = AppEditor { componentName : String, componentParams : Maybe Json.Encode.Value }
-    | PredefinedEditor String
+    | PredefinedEditor PredefinedEditorType
     | DefaultEditor
 
 
-type alias CellEditorParams =
-    { values : List String }
+{-| PredefinedEditorType
+
+  - `LargeTextEditor` will use the agLargeTextCellEditor
+  - `SelectionEditor` will use the agRichSelectCellEditor
+
+-}
+type PredefinedEditorType
+    = LargeTextEditor { maxLength : Int, rows : Int, cols : Int }
+    | SelectionEditor { values : List String }
```

Comment:
I think we should still provide the option to use any predefined editor from AgGrid by passing a string (e.g. agDateCellEditor, agNumberCellEditor, ...). It should not be limited to the ones we define as type. Especially since we didn't do that before.

And I think instead of defining a sub-type for the various editors, we could just merge them to the `CellEditor` type.

```elm
type CellEditor
    = AppEditor { componentName : String, componentParams : Maybe Json.Encode.Value }
    | LargeTextEditor { maxLength : Int, rows : Int, cols : Int }
    | SelectionEditor { values : List String }
    | PredefinedEditor String
    | DefaultEditor
```

That way we would also not have breaking changes.

Reply:
Sounds good!